internal/configs/transportserver.go

```diff
@@ -35,10 +35,34 @@ func generateTransportServerConfig(transportServerEx *TransportServerEx, listene
 	upstreams := generateStreamUpstreams(transportServerEx, upstreamNamer, isPlus)
 
 	var proxyRequests, proxyResponses *int
+	var connectTimeout, nextUpstreamTimeout string
+	var nextUpstream bool
+	var nextUpstreamTries int
 	if transportServerEx.TransportServer.Spec.UpstreamParameters != nil {
 		proxyRequests = transportServerEx.TransportServer.Spec.UpstreamParameters.UDPRequests
 		proxyResponses = transportServerEx.TransportServer.Spec.UpstreamParameters.UDPResponses
+
+		nextUpstream = transportServerEx.TransportServer.Spec.UpstreamParameters.NextUpstream
+		if nextUpstream {
+			nextUpstreamTries = transportServerEx.TransportServer.Spec.UpstreamParameters.NextUpstreamTries
+			nextUpstreamTimeout = transportServerEx.TransportServer.Spec.UpstreamParameters.NextUpstreamTimeout
```

Comment:
Would the following yaml
```yaml
upstreamParameters:
  connectTimeout: "60s"
  nextUpstream: true
  nextUpstreamTries: 1
```
  
Lead to the generation of the following invalid config?
```nginx
proxy_connect_timeout 60s;

proxy_next_upstream on;
proxy_next_upstream_timeout ; # optional in yaml, not optional in tmpl/config with no default?
proxy_next_upstream_tries 1;
```